server/plugin.ts

```diff
@@ -190,6 +191,9 @@ export class ObservabilityPlugin
           },
         },
       },
+      migrations: {
+        '3.0.0': migrateV1IntegrationToV2Integration,
```

Comment:
Not 100% sure I've set up migration correctly, messed with it a bit to get the type checking to work out but I'm not sure when this migration triggers.

Reply:
Hi
How would migration be handled an by whom ? 
For example if I have a migration script - where can I specify it ?

Reply:
Migrations run on OSD startup, it's handled in OSD and will apply to all of the installed templates that are on a lower version. But I'm not sure how OSD detects the version.

For content, I migrated all of the prebuilt content, but I'll need to update the bundle script for the new version.